src/Form/Activity/ActivityNewType.php

```diff
@@ -4,29 +4,39 @@
 
 use Symfony\Component\Form\Extension\Core\Type\MoneyType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 use Vich\UploaderBundle\Form\Type\VichImageType;
 
 class ActivityNewType extends ActivityEditType
 {
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
-        parent::buildForm($builder, $options);
```

Comment:
Since the calll to the parent method is completely omitted here now, and the configureOptions is overridden as well, is there still any merit to extending the ActivityEditType? Otherwise, I'd suggest to simply extending the default AbstractType

Reply:
Good point!